frontend/src/components/Loading/Loading.jsx

```diff
@@ -0,0 +1,20 @@
+import "./Loading.css";
+import { useContext } from "react";
+import { UserContext } from "../App/App";
+
+export default function Loading() {
+    const { loading } = useContext(UserContext);
+
+    return loading ? (
+        <div
```

Comment:
Can you explain how does the animation work? I couldn't find the code that makes it spin.

Reply:
It's bootstrap code, they give you class premade classes.